doc/modules/pipeline.rst

```diff
@@ -124,6 +119,43 @@ i.e. if the last estimator is a classifier, the :class:`Pipeline` can be used
 as a classifier. If the last estimator is a transformer, again, so is the
 pipeline.
 
+Caching transformers: avoid to repeat computation
+-------------------------------------------------
+
+.. currentmodule:: sklearn.pipeline
+
+Fitting transformers may sometimes computational expensive. The
+:class:`Pipeline` allows to cache the transformer after calling `fit`.
```

Comment:
"allows to" -> "can".

single backticks do nothing. use double backticks around fit


Actually I think it's better mentioning `memory` here: "With its memory parameter set, Pipeline will cache each transformer after calling fit"